language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -1910,4 +1911,65 @@ public static String getModifiedUri(WorkspaceManager workspaceManager, String ur
 
         return Optional.ofNullable(evalNode);
     }
+
+    /**
+     * Get the list of function arguments from the invokable symbol.
+     *
+     * @param symbol Invokable symbol to extract the arguments
+     * @param ctx    Lang Server Operation context
+     * @return {@link List} List of arguments
+     */
+    public static List<String> getFuncArguments(FunctionSymbol symbol, BallerinaCompletionContext ctx) {
```

Comment:
What if we make this function return `List<ParameterSymbol>` and write another util to get a list of modified type names?

Reply:
+1